doc/api/events.md

```diff
@@ -653,6 +653,46 @@ newListeners[0]();
 emitter.emit('log');
 ```
 
+## EventEmitter.once(emitter, name)
```

Comment:
Should not this be `events.once(emitter, name)`? Otherwise, this seems to be a static `EventEmitter` method like `EventEmitter.listenerCount()` above. If I understand correctly, this would be called as `events.once()` in the REPL.